cmd/admin-info.go

```diff
@@ -26,17 +39,191 @@ var adminInfoCmd = cli.Command{
 	Action: mainAdminInfo,
 	Before: setGlobalsFromContext,
 	Flags:  globalFlags,
-	Subcommands: []cli.Command{
-		adminInfoServer,
-		adminInfoCPU,
```

Comment:
Why are we removing this `adminInfoCPU`. With the present admin info PR https://github.com/minio/minio/pull/8497 , we don't fetch the CPU stats. Moreover the `adminInfoCPU,` uses `ServerCPULoadInfo()` , which is a different API all together. 

Same for `adminInfoMem`

Reply:
We are going back to the old naming convention. The command will simply be `mc admin info alias`